packages/http-client-csharp/package.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "@typespec/http-client-csharp",
+    "version": "0.2.0",
+    "author": "Microsoft Corporation",
+    "description": "The typespec library that can be used to generate C# models from a TypeSpec REST protocol binding",
+    "homepage": "https://github.com/Microsoft/typespec",
+    "readme": "https://github.com/Microsoft/typespec/blob/main/packages/http-client-csharp/readme.md",
+    "license": "MIT",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/Microsoft/typespec.git"
+    },
+    "bugs": {
+        "url": "https://github.com/Microsoft/typespec/issues"
+    },
+    "keywords": [
+        "typespec"
+    ],
+    "type": "module",
+    "main": "dist/src/index.js",
+    "scripts": {
```

Comment:
Looks like these scripts could be aligned with latest practices (e.g. vitest). Compare with a package.json in main.